src/libs/ReportUtils.js

```diff
@@ -1707,6 +1707,39 @@ function canLeaveRoom(report, isPolicyMember) {
     return true;
 }
 
+/**
+ * Returns display names for those that can see the whisper.
+ * However, it returns "you" if only the current user can see it besides the person that sent it.
+ *
+ * @param {string[]} participants
+ * @returns {string}
+ */
+function getWhisperDisplayNames(participants) {
+    const isOneOnOneWhisper = this.isOnlyVisibleByCurrentUser(participants);
+
+    // If we removed all users, it means it's only visible to "you"
+    if (isOneOnOneWhisper) {
```

Comment:
```suggestion
    // When the current user is the only participant, the display name needs to be "you" because that's the only person reading it
    if (isCurrentUserTheOnlyParticipant) {
```